src/archetypes/java-web-app/openshift/build.yaml

```diff
@@ -0,0 +1,198 @@
+---
```

Comment:
Should we update the hello-world-app to use the new java archetype pattern in the nrdk( and do away with its own version of build and deploy yamls), remove secrets and rather use webade jar

Reply:
I think eventually yes. As we are now trying to identify patterns, it is a fair assumption that hello-world would represent one of the patterns.

Reply:
Github tells me this file changed post the review i did last time but isnt showing what changed